pkg/handler/project/project.go

```diff
@@ -309,6 +309,84 @@ func (h *handler) GetMembers(c *gin.Context) {
 		&view.PaginationResponse{Pagination: query.Pagination, Total: total}, nil, nil))
 }
 
+// DeleteMember godoc
+// @Summary Delete member in a project
+// @Description Delete member in a project
+// @Tags Project
+// @Accept  json
+// @Produce  json
+// @Param Authorization header string true "jwt token"
+// @Param id path string true "member ID"
+// @Success 200 {object} view.MessageResponse
+// @Failure 400 {object} view.ErrorResponse
+// @Failure 404 {object} view.ErrorResponse
+// @Failure 500 {object} view.ErrorResponse
+// @Router /project/:id/members [delete]
+func (h *handler) DeleteMember(c *gin.Context) {
+	memberID := c.Param("id")
```

Comment:
this is projectID. MemberID can be embed in body

Reply:
because we dont need projectID here, the are any way for me to use employee ID in the url path

Reply:
updated